File: Minimum_Sum_of_Four_Digit_Number_After_Splitting_Digits.go
package main

import (
	"fmt"
	"sort"
	"strconv"
)

func main() {
	num := 2932
	fmt.Println(minimumSum(num))

}

func minimumSum(num int) int {
	num1 := []int{}
	for i := 0; i < 4; i++ {
		num1 = append(num1, num%10)
		num /= 10

	}
	sort.Ints(num1)
	var n1 string
	var n2 string

	for i := 0; i < len(num1); i++ {
		if i%2 == 0 {
			n1 += strconv.Itoa(num1[i])
		} else {
			n2 += strconv.Itoa(num1[i])
		}
	}

	n3 := 0
	if i, err := strconv.Atoi(n1); err == nil {
		n3 = i

	}

	n4 := 0

	if i, err := strconv.Atoi(n2); err == nil {
		n4 = i

	}

	return n3 + n4
}

File: Minimum_Sum_of_Four_Digit_Number_After_Splitting_Digits/Minimum_Sum_of_Four_Digit_Number_After_Splitting_Digits.py
num = "2932"
def minimumSum(num):
    n1=sorted(list(map(int,str(num))))
    s1,s2 ="",""
    for i in range(len(n1)):
        if i%2 ==0 :
            s1 += str(n1[i])
        else:
            s2 += str(n1[i])

    return (int(s1)+int(s2))
    


print(minimumSum(num))